RecoVertex/PrimaryVertexProducer/interface/DAClusterizerInZT_vect.h

```diff
@@ -33,34 +33,38 @@ class DAClusterizerInZT_vect final : public TrackClusterizerInZ {
 
       pi.push_back(new_pi);  // track weight
       Z_sum.push_back(1.0);  // Z[i]   for DA clustering, initial value as done in ::fill
+      kmin.push_back(0);
+      kmax.push_back(1);
     }
 
     unsigned int getSize() const { return z.size(); }
 
     // has to be called everytime the items are modified
     void extractRaw() {
-      z_ = &z.front();
-      t_ = &t.front();
-      dz2_ = &dz2.front();
-      dt2_ = &dt2.front();
-      Z_sum_ = &Z_sum.front();
-      pi_ = &pi.front();
+      _z = &z.front();
+      _t = &t.front();
+      _dz2 = &dz2.front();
+      _dt2 = &dt2.front();
+      _pi = &pi.front();
+      _Z_sum = &Z_sum.front();
     }
 
-    double *z_;   // z-coordinate at point of closest approach to the beamline
-    double *t_;   // t-coordinate at point of closest approach to the beamline
-    double *pi_;  // track weight
-
-    double *dz2_;    // square of the error of z(pca)
-    double *dt2_;    // square of the error of t(pca)
-    double *Z_sum_;  // Z[i]   for DA clustering
-
-    std::vector<double> z;                         // z-coordinate at point of closest approach to the beamline
-    std::vector<double> t;                         // t-coordinate at point of closest approach to the beamline
-    std::vector<double> dz2;                       // square of the error of z(pca)
-    std::vector<double> dt2;                       // square of the error of t(pca)
-    std::vector<double> Z_sum;                     // Z[i]   for DA clustering
-    std::vector<double> pi;                        // track weight
+    double *_z;   // z-coordinate at point of closest approach to the beamline
+    double *_t;   // t-coordinate at point of closest approach to the beamline
+    double *_pi;  // track weight
+
+    double *_dz2;    // square of the error of z(pca)
+    double *_dt2;    // square of the error of t(pca)
+    double *_Z_sum;  // Z[i]   for DA clustering
+
+    std::vector<double> z;      // z-coordinate at point of closest approach to the beamline
+    std::vector<double> t;      // t-coordinate at point of closest approach to the beamline
+    std::vector<double> dz2;    // square of the error of z(pca)
+    std::vector<double> dt2;    // square of the error of t(pca)
+    std::vector<double> Z_sum;  // Z[i]   for DA clustering
+    std::vector<double> pi;     // track weight
+    std::vector<unsigned int> kmin;
+    std::vector<unsigned int> kmax;
     std::vector<const reco::TransientTrack *> tt;  // a pointer to the Transient Track
```

Comment:
since this is touched already:
- please use trailing underscore, leading underscore is not allowed http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1
- please consistently use a trailing underscore for all member data so that it can be distinguished from local data in method implementation